stoiip_prob/main.py

```diff
@@ -1,6 +1,26 @@
+import numpy as np
+# from stoiip_prob import stoiip as stp
+from stoiip_prob.stoiip import stoiip
+
 #%% Define input variables
 h = 12
 A = 450
+poro = 0.15
+swc= 0.35
```

Comment:
Add appropriate spaces before and after assignations ("=")

Reply:
Done